task/build-helm-chart-oci-ta/0.2/build-helm-chart-oci-ta.yaml

```diff
@@ -291,13 +291,15 @@ spec:
           exit 1
         fi
 
-        # Extract digest from the skopeo copy output
-        digest=$(echo "$skopeo_output" | grep "Copying config" | awk '{print $3}' 2>/dev/null || echo "")
-        if [ -n "$digest" ]; then
-          echo "Successfully retrieved digest from skopeo copy: $digest"
-        else
-          echo "Could not retrieve digest from skopeo copy output"
+        # Get the manifest digest of the pushed chart
+        # The manifest digest is the SHA256 of the entire manifest JSON
+        if ! manifest_digest=$(skopeo inspect --raw "docker://$pushed" | sha256sum | awk '{print "sha256:" $1}' 2>/dev/null); then
```

Comment:
it seems we mgiht be able to do this directly with curl without locally calcluating the digest. See:

```
curl -s -H "Accept: application/vnd.oci.image.manifest.v1+json" \
  https://quay.io/v2/redhat-user-workloads/konflux-vanguard-tenant/caching/squid-helm/manifests/0.1.140_2679b7c-on-pr | jq .
```